### PR TITLE
[ARM-5353] Add some pictograms

### PR DESCRIPTION
**Pull request by hweeTan**
[ARM-5353]

### Description

Add the missing pictograms to use in Client Portal `/lobby` route

### How to test

- Check storybook for the new pictograms:
  - Clock2Blue64
  - Clock2White64
  - ReportBlue64
  - ReportWhite64
  - HandDeliveryBlue64
  - HandDeliveryWhite64
  - TeamBusinessBlue64
  - TeamBusinessWhite64
  - TeamOkrsBlue64
  - TeamOkrsWhite64

The names are from [the Figma file](https://www.figma.com/design/6UPpJrQbP02OqBjtGL6KTr/Pictograms?node-id=1151-322&t=spuu28O6oMikwUFS-1) 

### Screenshots

| Name.                                 | Screenshot.                                  |
| --------------------------------------- | --------------------------------------- |
| Report | image |
| HandDelivery | image |
| TeamBusiness | image |
| TeamOkrs | image |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- N/A Annotate all `props` in component with documentation
- N/A Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- N/A Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- N/A Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)

**Breaking change**

- N/A codemod is created and showcased in the changeset
- N/A test alpha package of Picasso in StaffPortal
- [x] Test alpha package in Client Portal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[ARM-5353]: https://toptal-core.atlassian.net/browse/ARM-5353?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hweeTan** [comment]
@toptal-bot run package:alpha-release

**Comment by hweeTan** [comment]
@toptal-anvil ping reviewers

File: .changeset/calm-toes-deliver.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+'@toptal/picasso-pictograms': minor
+---
+
+Add a couple pictograms:
```
Comment (sashuk) [inline .changeset/calm-toes-deliver.md:5]: NIT

```suggestion
- add pictograms:
```

**Review by sashuk — APPROVED** [review]
Tested locally, code looks good


Screenshot 2024-06-12 at 10 47 08

**Comment by sashuk** [comment]
@toptal-anvil ping reviewers

**Review by Q-Man — APPROVED** [review]
Nice. 👍

**Pull request merged**